pages/common/lsar.md

```diff
@@ -0,0 +1,29 @@
+# lsar
+
+> List archive file contents.
+> See also: `unar`, `ar`.
+> More information: <https://manned.org/lsar>.
+
+- List archive file contents:
+
+`lsar {{archive}}`
+
+- List password protected archive file contents:
+
+`lsar {{archive}} --password {{password}}`
+
+- Print al[L] available information about each file in the archive (it's very long):
+
+`lsar {{-L|--verylong}} {{archive}}`
+
+- Test the integrity of the files in the archive (if possible):
+
+`lsar --test {{archive}}`
+
+- List archive file contents in JSON format:
+
+`lsar --json {{archive}}`
+
+- Display help information:
+
+`lsar --help`
```

Comment:
@gutjuri 
The [unar](https://github.com/tldr-pages/tldr/blob/main/pages/common/unar.md) also uses `{{archive}}` instead of `{{path/to/archive}}`, so if we were to change that, we would have to do it in both two commands.

Reply:
Right, would you like to do it? If yes, you could do it in this PR. If not, I can submit a PR later on. However you like :)